config.yaml

```diff
@@ -18,6 +18,11 @@ disease:
     path: ontology-inputs
     owl_jq: '.["@graph"][] | select (.["@type"] == ("owl:Class","owl:Restriction") and ."@id" != null)|.subClassOf=([.subClassOf]|flatten|del(.[]|nulls))|.hasExactSynonym=([.hasExactSynonym]|flatten|del(.[]|nulls))|.hasDbXref=([.hasDbXref]|flatten|del(.[]|nulls))|.inSubset=([.inSubset]|flatten|del(.[]|nulls))|.hasAlternativeId=([.hasAlternativeId]|flatten|del(.[]|nulls))|@json'
     diseases_static_file: diseases_efo.jsonl
+    url_prefix_mapping:
+      EFO: http://www.ebi.ac.uk/efo/
```

Comment:
We could probably use [identifiers.org](https://identifiers.org) for this in the future, I know _EFO_ is a registered namespace there, so EFO_0123456 -> https://identifiers.org/EFO:0123456
_Orphanet_ is also registered, but MONDO is missing.
Would you mind exploring this option with OLS team (registering MONDO in identifiers.org) and then using the [identifiers.org](https://identifiers.org) _resolution services_?